src/search.cpp

```diff
@@ -398,7 +399,8 @@ Workload Search::refine(Workload wl) {
         if (target_queues.find(q) == target_queues.end()) {
             Indiv n_indiv = Indiv(metric_t::CENQ, q);
             Comp n_wl_spec = Comp(n_indiv, op_t::LE, 0u);
-            wl.add_spec(TimedSpec(n_wl_spec, total_time, total_time));
+            WlSpec* n_wl_spec_ptr = new Comp(n_wl_spec);
+            wl.add_spec(TimedSpec(n_wl_spec_ptr, total_time, total_time));
```

Comment:
Same comment as [this](https://github.com/all-things-networking/fperf/pull/27/files#r1484920823)

Reply:
[Search handles pointers more elegantly](https://github.com/all-things-networking/fperf/pull/27/commits/00fe78d47212573296f3abb7df4352785a85d2f3)